layers/+lang/c-c++/packages.el

```diff
@@ -61,7 +61,10 @@
 
 (defun c-c++/init-clang-format ()
   (use-package clang-format
-    :if c-c++-enable-clang-support))
+    :if c-c++-enable-clang-support
+    :init
+    (when c-c++-enable-clang-format-on-save
+      (add-hook 'c-mode-common-hook 'clang-format/format-on-save))))
```

Comment:
Should this add hook only for c/c++ mode as follows inline with rest of layer?

`(spacemacs/add-to-hooks 'clang-format/format-on-save '(c-mode-hook c++-mode-hook))`